torch/_subclasses/fake_tensor.py

```diff
@@ -784,9 +789,15 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             return converter(self, args[0])
 
         if self.check_for_non_fake(flat_arg_tensors):
-            raise Exception(
-                "Invoking operators with non-Fake Tensor inputs in FakeTensorMode is not yet supported. "
-                f"Please convert all Tensors to FakeTensors first. Found in {func}(*{args}, **{kwargs})"
+            if (torch.Tag.inplace_view in func.tags) or (not self.allow_non_fake_inputs):
+                raise Exception(
+                    "Invoking operators with non-Fake Tensor inputs in FakeTensorMode is not yet supported. "
+                    f"Please convert all Tensors to FakeTensors first. Found in {func}(*{args}, **{kwargs})"
```

Comment:
Can we update the error message now that this is supported and to suggest `allow_non_fake_inputs` ? And have a separate error msg for the `torch.Tag.inplace_view` case